pinot-core/src/main/java/org/apache/pinot/core/segment/index/readers/ConstantValueDoubleDictionary.java

```diff
@@ -19,20 +19,25 @@
 package org.apache.pinot.core.segment.index.readers;
 
 /**
- * Dictionary for constant-value double
+ * Dictionary of a single double value.
```

Comment:
How about constant double value? I also used "single double" earlier but was persuaded by Subbu it sounded a little weird

Reply:
I prefer the current comment because the dictionary only contains a single value, instead of multiple values of the same value. @mcvsubbu Opinion?
Future plan would be wrap all these classes (forward index, inverted index, dictionary for default column) into one single class.

Reply:
I like the one class approach, and had suggested that earlier to Haibo. Let us not worry too much about a comment